packages/relay/src/lib/clients/sdkClient.ts

```diff
@@ -113,7 +113,7 @@ export class SDKClient {
         this.consensusNodeClientHistorgram = new Histogram({
             name: metricHistogramName,
             help: 'Relay consensusnode mode type status cost histogram',
-            labelNames: ['mode', 'type', 'status', 'caller', 'interactingEntity'],
+            labelNames: ['mode', 'type', 'status', 'caller', 'interactingEntity', 'type'],
```

Comment:
we already have a type label, if we were to use the same metric, we should reuse the type label, and registered as gas.